scripts/setup_linux.sh

```diff
@@ -1,61 +1,254 @@
-#!/bin/bash
-# Make sure that conda is available
+#!/usr/bin/env bash
+# This script installs garage on Linux distributions
```

Comment:
Please add a note to maintainers to update the dockerfiles in docker/ if they change this file (and vice-versa)